src/components/Nav/Nav.js

```diff
@@ -1,7 +1,110 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import HomeLogo from '../../assets/images/nav/home-logo.svg';
+import Basket from '../../assets/images/nav/shopping-bag.png';
+import Search from '../../assets/images/nav/search.png';
+import './Nav.scss';
 
-const Nav = () => {
-  return <div>Nav</div>;
+const Nav = props => {
+  const navigate = useNavigate();
+  const { setSearchKeyword, basket, setSelectedCategory } = props;
+  const [inputSearchVal, setInputSearchVal] = useState('');
+  let isLogined = localStorage.getItem('signup_token') ? true : false;
```

Comment:
```suggestion
  const loginToken = localStorage.getItem('signup_token');
```
위와 같이 구성하면 있을 때는 해당 토큰의 문자가 loginToken에 담기게 되고
없다면 undefined가 나와서 자동으로 false처럼 쓰여지게 될 거에요!